tests/network/user_defined_network/test_user_defined_network.py

```diff
@@ -147,7 +147,7 @@ def test_basic_connectivity_between_udn_vms(self, vma_udn, vmb_udn_non_migratabl
     @pytest.mark.gating
     @pytest.mark.xfail(
         reason=f"{QUARANTINED}: Flaky test, fails on connecting to VM console; tracked in CNV-66779",
-        run=False,
+        run=True,
```

Comment:
I suggest removing this line for consistency.
```suggestion
```
In the repository, there are other `xfail` cases without the `run` parameter, and none use `run=True`.

Reply:
I guess what you mean to say, in much simpler (and fewer) words, is that `run=True` is the default. The fact that no other call to xfail in the repo sends it is not relevant.
But is it indeed the default? @rnetser I removed the explicit sending of `run=True` according to @RoniKishner comment, and verified that the behavior is as I expect (i.e. that the test is executed). However, I am looking at the [xfail implementation](https://github.com/pytest-dev/pytest/blob/main/src/_pytest/mark/structures.py#L503), and I see that `run` has no default value. Am I missing something? Because to me it seems that I do need to explicitly send `run=True`. WDYT?

Reply:
/build-and-push-container